must.go

```diff
@@ -3,11 +3,13 @@ package must
 // TestingT is an interface wrapper around *testing.T
 type TestingT interface {
 	Errorf(format string, args ...interface{})
+	Helper()
 }
 
 // MustTester defines an interface with functions matching the package level check functions, without the requirement to specify a TestingT.
 type MustTester interface {
 	BeEqual(expected, got interface{}, a ...interface{}) bool
 	BeEqualErrors(expected, got error, a ...interface{}) bool
+	BeNoError(got error, a ...interface{}) bool
```

Comment:
Weren't you planning on allowing a bool to be passed here to verify whether or not an error was expected?

Reply:
That's coming up. This was just adding the already existing function to the interface.

Reply:
Awesome, thanks for catching the missing item!